openedx/core/djangoapps/edx_global_analytics/tasks.py

```diff
@@ -78,14 +84,28 @@ def count_data():
         platform_name = settings.PLATFORM_NAME
     else:
         platform_name = Site.objects.get_current()
- 
+
+    # Sending instance data
+    paranoid = {
```

Comment:
i think we can just name it data, and paranoid will be in the settings, and there will be no need to copy